app/geo/gui/src/ts/twxplore/gui/geo/reducerFunctions/updateAttributeStatesOfFeatureType.ts

```diff
@@ -9,42 +9,42 @@ export const updateAttributeStatesOfFeatureType = (
   addedFeature: MapFeature
 ) => {
   //Loop through the feature's attribute
-  for (const attribute of Object.keys(addedFeature)) {
+  for (const attributeName of Object.keys(addedFeature)) {
+    const attributeKey = attributeName as keyof MapFeature;
     //If the arttribute is numeric
-    if (getFeatureAttributeByName(attribute).isNumeric) {
+    if (getFeatureAttributeByName(attributeName).isNumeric) {
       //If this is the first time coming across this attribute for this feature type
       if (
-        !attributeStatesOfFeatureType[attribute].min &&
-        !attributeStatesOfFeatureType[attribute].min
+        !attributeStatesOfFeatureType[attributeName].min &&
```

Comment:
Still not handling the case where min and/or max could be 0. Should be typeof(...) === "undefined".

Reply:
I initialize all possible combinations of  attributeStatesOfFeaatureType[attribute].min/max to null in initialMapStates.tsx so it can never undefined.
If min/max is 0 then that means this is not the first time coming across the attribute, because a feature must have had 0 for transmissionPower for example, and that's what changed min/max to 0

Reply:
My point remains. !0 and !null will both be true in this branch.

Reply:
Checked for null by using === equality check instead of !attributeStates...